### DISCUSSION TITLE
[Live Diarization] The first few words of the 2nd speaker consistently gets tagged to the first speaker.

### DISCUSSION THREAD

**Discussion by vincentpaca** [deepgram/community#614] [General help]
### Which Deepgram product are you using?

Deepgram API

### Details

Hi,

When diarizing streamed audio, I've always had the first few words (1-3) of my 2nd speaker tagged as the first speaker. This issue has been consistent, and getting repeatable results with my audio samples.

I've tried increasing the silence between the speakers but still, the first few words of the second speaker gets tagged as the first speaker.

The results have been the same with nova-2, base, nova-2-phonecall.

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

wss://api.deepgram.com/v1/listen?model=nova-2&diarize=true&interim_results=true&smart_format=true

### If possible, please attach an example audio file to reproduce the issue.

https://drive.google.com/file/d/1NzB1WRJzXq5vNUwyCtu33TiaHJkr6kwZ/view?usp=drive_link

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
